boards/rp-pico/examples/pico_spi_sd_card.rs

```diff
@@ -39,37 +39,6 @@
 //!
 //!     $ mkfs.fat /dev/sdj1
 //!
-//! In the following ASCII art the SD card is also connected to 5 strong pull up
-//! resistors. I've found varying values for these, from 50kOhm, 10kOhm
-//! down to 5kOhm.
-//! Stronger pull up resistors will eat more amperes, but also allow faster
-//! data rates.
-//!
-//! ```text
-//!                    +3.3V
-//!          Pull Ups ->||||
-//!                 4x[5kOhm]
-//!                     ||| \
-//!  _______________    |||  \
-//! |     DAT2/NC  9\---o||   \                            _|USB|_
-//! | S   DAT3/CS   1|---o+----+------SS--\               |1  R 40|
-//! | D   CMD/DI    2|----o----+-----MOSI-+-\             |2  P 39|
-//! |     VSS1      3|-- GND   |          | |         GND-|3    38|- GND
-//! | C   VDD       4|-- +3.3V |  /--SCK--+-+----SPI0 SCK-|4  P 37|
-//! | A   CLK/SCK   5|---------+-/        | \----SPI0 TX--|5  I 36|- +3.3V
-//! | R   VSS2      6|-- GND   |  /--MISO-+------SPI0 RX--|6  C   |
-//! | D   DAT0/DO   7|---------o-/        \------SPI0 CSn-|7  O   |
-//! |     DAT1/IRQ  8|-[5k]- +3.3V                        |       |
-//!  """"""""""""""""                                     |       |
-//!                                                       |       |
-//!                                                       .........
-//!                                                       |20   21|
-//!                                                        """""""
-//! Symbols:
-//!     - (+) crossing lines, not connected
-//!     - (o) connected lines
-//! ```
-//!
```

Comment:
Why do you remove the schematic?

Reply:
Ok, that makes sense. I like examples to describe the necessary cabling, but without the resistors, there's not much left in the diagram.

Also, I think the relationship between pull-ups and data rate hold for I2C, but not for SPI, which is actively driven. One more reason to remove this comment.